cmd/entrypoint.go

```diff
@@ -24,10 +24,13 @@ func init() {
 	RootCmd.AddCommand(entrypointCmd)
 	entrypointCmd.Flags().BoolVarP(&execCommand, "exec", "e", false, "Delegate to the command directly with exec(3), no cleanup")
 	entrypointCmd.Flags().StringSliceVarP(&filesToDecrypt, "files", "f", []string{}, "files to decrypt (default all)")
+	entrypointCmd.Flags().BoolVarP(&allowFail, "allow-fail", "a", false, "Do not fail if not all files can be decrypted")
```

Comment:
You might add a comment here about how this `&allowFail` also is set on the decrypt command, which is how this gets passed through.